deployment/mcad-controller/templates/manager/deployment.yaml

```diff
@@ -16,6 +16,8 @@ spec:
     spec:
       securityContext:
         runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
We could add this to values.yaml if you think being able to configure it is important.

Reply:
I suggest flipping the bit in `config/manager/manager.yaml` manager as well.